coverage/PENDING_SUBMIT/amdvlk/0518-COVERAGE-optional-h-452/cov-color-output-undefined-in-unexecuted-branch.amber

```diff
@@ -0,0 +1,410 @@
+#!amber
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# A test for a coverage-gap found by the GraphicsFuzz project.
+
+# Short description: A fragment shader that covers specific LLVM code paths
+
+# The test passes because the shader always writes red.
+
+SHADER vertex variant_vertex_shader PASSTHROUGH
+
+# variant_fragment_shader is derived from the following GLSL:
+# #version 320 es
+# #define _int_20 _GLF_uniform_int_values[0]
+# #define _int_1 _GLF_uniform_int_values[1]
+# #define _int_0 _GLF_uniform_int_values[2]
+# #define _int_2 _GLF_uniform_int_values[3]
+# #define _uint_60433 _GLF_uniform_uint_values[0]
+# #define _float_1_0 _GLF_uniform_float_values[0]
+# #define _float_0_0 _GLF_uniform_float_values[1]
+# #define _float_1_7 _GLF_uniform_float_values[2]
+#
+# precision highp float;
+#
+# precision highp int;
+#
+# // Contents of _GLF_uniform_uint_values: 60433
+# layout(set = 0, binding = 0) uniform buf0
+# {
+#     uint _GLF_uniform_uint_values[1];
+# };
+# // Contents of _GLF_uniform_float_values: [1.0, 0.0, 1.7]
+# layout(set = 0, binding = 1) uniform buf1
+# {
+#     float _GLF_uniform_float_values[3];
+# };
+# // Contents of _GLF_uniform_int_values: [20, 1, 0, 2]
+# layout(set = 0, binding = 2) uniform buf2
+# {
+#     int _GLF_uniform_int_values[4];
+# };
+# // Contents of three: 3
+# layout(set = 0, binding = 3) uniform buf3
+# {
+#     int three;
+# };
+#
+# layout(location = 0) out vec4 _GLF_color;
+#
+# void func0()
+# {
+#     vec4 tmp;
+#     // Always true.
+#     if(gl_FragCoord.x > _float_0_0)
+#     {
+#         tmp = _GLF_color;
+#     }
+#     _GLF_color = tmp;
+# }
+#
+# int func1()
+# {
+#     int a = _int_1;
+#
+#     while(a < _int_2)
+#     {
+#         // Always true.
+#         if(three > _int_1)
+#         {
+#             func0();
+#             a = _int_2;
+#         }
+#         else
+#         {
+#             func0();
+#         }
+#     }
+#
+#     return a;
+# }
+#
+# void main()
+# {
+#     // Always true.
+#     if(gl_FragCoord.x > _float_0_0)
+#     {
+#         _GLF_color = vec4(_float_1_0, _float_0_0, _float_1_0, _float_1_7);
+#     }
+#     else
+#     {
+#         _GLF_color = unpackSnorm4x8(_uint_60433);
+#     }
+#
+#     int a = _int_0;
+#     for(int i = 0; i < 5; i ++)
+#     {
+#         for(int j = 0; j < 2; j ++)
+#         {
+#             a += func1();
```

Comment:
Comment that this is his 10 times, and that func1() always returns 2, hence a ends up with the value 20.